public/javascripts/gformFileUpload.js

```diff
@@ -301,7 +297,7 @@
             return checkConfirmation(formTemplateId, key);
           }, onError)
           .then(function(response) {
-            if(response == "") {
+            if(response === undefined) {
```

Comment:
Under what condition is response `undefined`?

Reply:
If the file is uploaded successfully, the `check` method returns `NoContent`. it means `response` is undefined.
https://github.com/hmrc/gform-frontend/blob/dg/GFORMS-1550/app/uk/gov/hmrc/gform/upscan/UpscanController.scala#L117